File: 2nd year/Laboratories/MPP/project/data/src/main/java/com/alex/mpp/data/domain/Result.java
package com.alex.mpp.data.domain;

import lombok.Data;
import lombok.NonNull;

import java.io.Serializable;

@Data
public class Result implements Serializable {
    private @NonNull Integer id;
    private @NonNull Integer gameId;
    private @NonNull Integer athleteId;
    private @NonNull Float value;
}

File: 2nd year/Laboratories/MPP/project-cs/project/domain/Result.cs
namespace project.domain
{
    public class Result
    {
        public int Id { get; set; }
        public int GameId { get; set; }
        public int AthleteId { get; set; }
        public double Value { get; set; }

        public override string ToString()
        {
            return
                $"{nameof(Id)}: {Id}, {nameof(GameId)}: {GameId}, {nameof(AthleteId)}: {AthleteId}, {nameof(Value)}: {Value}";
        }
    }
}